models/prices/aptos/prices_aptos_tokens.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+        schema='prices_aptos',
+        alias = 'tokens',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static', 'dunesql']
+        )
+}}
+SELECT 
+    token_id
+    , blockchain
+    , symbol
+    , from_base58(contract_address) as contract_address
```

Comment:
Hi @chef-seaweed, we need to make sure this column is `varbinary` type.
Do you have any suggestion or idea?

(In solana spell we use `from_base58` to handle this column, but seems it does not work at here.)

Reply:
Yeah  seems like we don't have a proper function to convert Aptos token address to `varbinary` type yet. 
Also we'll have `Error: Not a hexadecimal digit: ":"` with `from_hex`
Should we build a dedicated price table for Aptos?

Reply:
Let me think if we need a new udf here.